api/package.json

```diff
@@ -36,6 +36,7 @@
     "config": "^3.3.6",
     "helmet": "^4.6.0",
     "jsonapi-serializer": "^3.6.7",
+    "mapshaper": "^0.5.62",
```

Comment:
I'd say this is not required in the API side, since we are not (yet) using it, but its being downloaded and used in your magical script 😃

Reply:
and I strongly suspect that the changes to the yarn.lock are not just this package, so review those too please